metricbeat/module/windows/service/service_integration_windows_test.go

```diff
@@ -5,9 +5,17 @@ package service
 import (
 	"testing"
 
+	"github.com/StackExchange/wmi"
+	"github.com/stretchr/testify/assert"
+
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 )
 
+type Win32Service struct {
+	Name      string
+	ProcessId uint32
+}
```

Comment:
Awesome! Can you expand this to validate some of the other parameters we collect.

Reply:
The only thing we can validate is `state`. `DisplayName` is different for some services. I think this is a bug of the wmi query
![image](https://user-images.githubusercontent.com/19551763/33162867-0d8dadf4-d02c-11e7-9e03-c938139ab236.png)